File: examples/LayaAir/src/proto/ReqChatGm.ts
namespace proto {
export class ReqChatGm
{
	private _content: string;


	public Encode(): net.Packet {
		let packet: net.Packet = new net.Packet();
		packet.WriteString(this._content);
		packet.Encode(1530);
		return packet;
	}


	public get content(): string { return this._content; }
	public set content(value: string) { this._content = value; }
}
}

File: examples/golang/client/proto/req_chat_gm.go
package proto

import (
	"packet"
)

type ReqChatGm struct {
	Content                  string
}

func (this *ReqChatGm) Encode() []byte {
	pack := packet.NewWriteBuff(64)

	pack.WriteString(this.Content)

	return pack.Encode(P_REQ_CHAT_GM)
}
